packages/haul-core/src/webpack/plugins/PreloadModulesDllPlugin.ts

```diff
@@ -98,13 +116,13 @@ export default class InitCoreDllPlugin {
               ...args: any[]
             ) => {
               tap.fn(compilation, ...args);
-              const initCoreDllModuleFactory = new InitCoreDllModuleFactory(
+              const initCoreDllModuleFactory = new PreloadDllModuleFactory(
                 () => {
                   const setupFilesModulesIds = compilation.modules.reduce(
```

Comment:
rename to just `modulesIds`? `setupFile` is not a thing now, so it may be confusing